manifest_prod.yml

```diff
@@ -1,12 +1,16 @@
 ---
 memory: 512M
 stack: lucid64
+instances: 2
+env:
+  NEW_RELIC_CONFIG_FILE: newrelic.ini
+  NEW_RELIC_ENV: staging
 applications:
 - name: api
   path: .
   host: fec-prod-api
   env:
-    PRODUCTION: True
+    FEC_API_DEBUG: False
```

Comment:
Thanks for catching that. Agree we should work on a standard solution. `configargparse` looks like it has a lot of command-line oriented stuff we probably don't need-- what do you think about https://github.com/mbr/flask-appconfig, especially the "ENV-only" mode?